content/SDKs/javascript/index.mdx

```diff
@@ -36,6 +36,108 @@ curl -fsSL https://bun.sh/install | bash
 
 For more information, visit the [Bun installation documentation](https://bun.sh/docs/installation).
 
+## Welcome Function
+
+The SDK allows you to customize the initial appearance of DevMode when it starts interacting with your agents by exporting a `welcome()` function. This function returns an `AgentWelcomeResult` object that includes a welcome message and optional example prompts.
+
+### AgentWelcomeResult Interface
+
+```typescript
+export interface AgentWelcomeResult {
+  /**
+   * The welcome prompt to display to the user
+   */
+  welcome: string;
+  /**
+   * The example prompts to display to the user
+   */
+  prompts?: {
+    /**
+     * The data in base64 format
+     */
+    data: string;
+    /**
+     * The data format
+     */
+    contentType: string;
+  }[];
+}
+```
+
+### Example Implementation
+
+```typescript
+// agent.ts
+import type { AgentHandler, AgentWelcomeResult } from '@agentuity/sdk';
+
+export const welcome = (): AgentWelcomeResult => {
+  return {
+    welcome: "Welcome to my Agent! How can I help you today?",
+    prompts: [
+      {
+        data: Buffer.from("What can you do?").toString("base64"),
```

Comment:
The buffer.from need to convert to base64 is a temp. need and will be changed in the next day or so.